testing/e2e/src/components/AudioGenUI.tsx

```
@@ -0,0 +1,114 @@
+import { useState } from 'react'
+import type { AudioGenerationResult } from '@tanstack/ai'
+import { generateAudioFn } from '@/lib/server-functions'
+import type { Feature, Mode, Provider } from '@/lib/types'
```

ESLint

[error] `@/lib/types` type import should occur before import of `@/lib/server-functions`

(import/order)